src/servers/sapphire_zone/Actor/InstanceObject.cpp

```diff
@@ -33,7 +34,7 @@ void Core::Entity::InstanceObject::setState( uint8_t state )
 {
    m_state = state;
 
-   m_parentInstance->updateInstanceObj( InstanceObjectPtr( this ) );
+   //m_parentInstance->updateInstanceObj( InstanceObjectPtr( this ) );
```

Comment:
intentional?

Reply:
yeah it crashed and i couldn't be bothered to figure out why lmao

Reply:
need to change how it works anyway cause i'm not a fan of the cyclic dependency anyway